docs/building-images.md

```diff
@@ -198,6 +218,18 @@ kubernetes:
 * `manifests` - Optional; manifests to apply to the cluster.
   Can be used separately or in combination with the [configuration directory](#kubernetes-1).
   * `urls` - Optional; list of HTTP(s) URLs to download the manifests from
+* `helm` - Optional; Helm charts to be deployed to the cluster.
+  * `charts` - Optional; Defines a list of Helm charts and configuration for each Helm chart.
```

Comment:
Is the plan that this is actually optional? I feel like validation would see if the section was created but no charts defined, unless we explicitly want to allow repositories to be defined without a chart installed.